app/assets/javascripts/hyrax/collections.js

```diff
@@ -10,6 +10,13 @@ Blacklight.onLoad(function () {
       form.append('<input type="hidden" value="add" name="collection[members]"></input>');
   });
 
+  // background banner image uploaded for branding  
```

Comment:
I'd appreciate this go into it's own file for code related to branding collections.  The existing `collections.js` is becoming a bit of a junk-drawer.

Reply:
I didn't write this code. It was already there when I started work on the project. That said I can move it into it's own file. No problem.

Reply:
Also, I didn't add any new JS to this file. I just changed the name of a CSS class (used as a selector) in pre-existing JS code.

Reply:
I don't feel comfortable adding the JS test since I didn't write the JS in question.

That said, I don't believe the "bg banner" JS is needed. I could just rewrite the show code to display the inline style directly and then remove the "bg banner" JS from the collections.js file. Seems like a win-win. How does that sound?

Reply:
✅
I updated the collections show code to display the inline style directly. As such, I removed the redundant JS code you were referring to. No need for a new file at this time. Though I will make a note of your concerns for future reference.